src/components/routes/Host/ListingForm/ListingFormNav/ListingFormNav.tsx

```diff
@@ -63,18 +62,18 @@ const ListingFormNav = ({ errors, history, id, isValid, onSubmit, setNextCrumb,
       <GeneralWrapper width={976}>
         <Prompt
           when={showAlert}
-          message={`Listing has unsaved changes ${!isValid ? `due to the following errors: ${JSON.stringify(Object.values(errors), null, 4)}`: ''}. Are you sure you want to proceed?`}>
+          message={`Listing has unsaved changes ${!formikProps.isValid ? `due to the following errors:\n\n${Object.values(formikProps.errors).join('\n').toString()}\n\n`: ''} Are you sure you want to proceed?`}>
```

Comment:
can we put this into a function? ``formatListingErrorsAlert(formikProps.errors)`` or something similar?

Reply:
made new function on bottom